SimPEG/Maps.py

```diff
@@ -14,11 +14,13 @@
 import scipy.sparse as sp
 from scipy.sparse.linalg import LinearOperator
 from scipy.interpolate import UnivariateSpline
+from scipy.spatial import cKDTree
```

Comment:
is there a map that now depends on this? it doesn't look like any maps were added or changed. If not, can you please remove the additional import